mesonbuild/compilers/c.py

```diff
@@ -1156,6 +1156,10 @@ def get_linker_always_args(self):
             return basic + ['-Wl,-headerpad_max_install_names']
         return basic
 
+    def get_symbol_export_file_args(self, path):
+        if self.compiler_type.is_osx_compiler:
```

Comment:
Does GCC support `-exported_symbols_list` on macOS? (Be careful, GCC on Linux interprets `-exported_symbols_list` as the `-e` flag with the value `xported_symbols_list`)